client/cli/src/commands/run_cmd.rs

```diff
@@ -250,6 +255,18 @@ pub struct RunCmd {
 		conflicts_with_all = &[ "sentry", "public-addr" ]
 	)]
 	pub sentry_nodes: Vec<MultiaddrWithPeerId>,
+
+	/// Run a temporary node.
+	///
+	/// A random temporary directory will be created to store the configuration and will be deleted
```

Comment:
"random temporary" => "temporary"

Please also make the user aware that this is the "base-path" and what will be part of this path. Database, node key etc.

Reply:
done in https://github.com/paritytech/substrate/pull/6221/commits/55ed555d4965f615a702b2ce991023eb4bc49fd5